### PR TITLE
New contact slice

### PR DESCRIPTION
**Pull request by pataruco**
### Motivation

Marketing asked us to change the contact us form to an email to link to hello@red-badger.com

### Test plan
Please go to [https://www-staging.red-badger.com/1573871/](https://www-staging.red-badger.com/1573871/), scroll down to the contact us slice and you will see the new slice as per designs 

- [Small](zpl.io/a7KMvj8)
- [Mediums](https://zpl.io/V4PW9XJ)
- [Large](https://zpl.io/ag939oZ)
### Pre-merge checklist

- [ ] Documentation
- [ ] Unit tests
- [ ] Reviews
  - [ ] Code review
  - [ ] Design review
- [ ] Manual testing
  - [ ] Windows 7 IE 11
  - [ ] Windows 10 Edge
  - [ ] Mac OS El Capitan Safari
  - [ ] Mac OS El Capitan Chrome
  - [ ] Mac OS El Capitan Firefox
  - [ ] iOS 9 Safari
  - [ ] Android 6 Chrome
  - [ ] Listen to the site on a screen reader
  - [ ] Navigate the site using the keyboard only
- [ ] Tester approved

### CODE REVIEW

**Comment by markholland** [comment]
Hover doesn't work for me on firefox

File: site/slices/new-contact-us-slice/index.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    };
+  }
+
+  onHover = () => {
+    this.setState({
+      isHovered: true,
+    });
+  };
+
+  onBlur = () => {
```
Comment (davidbasalla) [inline site/slices/new-contact-us-slice/index.js]: Is `onBlur` the appropriate name here? `offHover` maybe? hmm I'm not good with the names today... 😄

File: site/pages/home/index.js [hunk 2]
```diff
@@ -6,7 +6,7 @@ import CaseStudyOverview from '../../components/case-study-overview';
 import Brie from './brie-slice';
 import TechSlice from '../../slices/tech-slice';
 import BlogSlice from './blog-slice';
-import ContactUs from '../../slices/contact-us-slice';
+import NewContactUs from '../../slices/new-contact-us-slice';
```
[the remark this answers is not in the document]
Comment (pataruco) [reply site/pages/home/index.js]: Cool

File: site/pages/home/index.js [hunk 3]
```diff
@@ -17,7 +17,7 @@ export type HomePageProps = {
   featuredBlogPosts: Array<Object>,
 };
 
-const HomePage = ({ contactUsURL, featuredBlogPosts }: HomePageProps) => {
+const HomePage = ({ featuredBlogPosts }: HomePageProps) => {
```
[the remark this answers is not in the document]
Comment (pataruco) [reply site/pages/home/index.js]: Yes, the `contactUSUrl`  was the URL given to the previous form  to do the POST

**Comment by pataruco** [comment]
Hi @markholland and @davidbasalla I made the changes you suggested, please could you review it again ▶️ [here](https://www-staging.red-badger.com/607b46a/)

File: site/slices/new-contact-us-slice/index.js (hunk 1 above)
Comment (markholland) [reply site/slices/new-contact-us-slice/index.js]: @davidbasalla The onBlur event is native to HTML

File: site/pages/home/index.js (hunk 3 above)
[the remark this answers is not in the document]
Comment (davidbasalla) [reply site/pages/home/index.js]: Okay, let's remove it then

**Comment by markholland** [comment]
New requirement has come in to add GA to the CTA. @x4d3 is doing it

**Comment by x4d3** [comment]
@markholland I've added the GA tracking.

**Pull request merged**